apps/docs/src/common/pages/resources/accessibility.tsx

```diff
@@ -8,7 +8,7 @@ const Page: FC<PageProps> = ({ location }) => (
   <div className="govuk-grid-row">
     <div className="govuk-grid-column-one-third">
       <aside>
-        <h2>Resources</h2>
+        <h2>Get started</h2>
```

Comment:
Why has this changed?

Reply:
No reason. I was experimenting and forgot to undo changes. I've updated this to remove second h2 now.